gui/app/simple_clock.cpp

```diff
@@ -0,0 +1,61 @@
+#include "simple_clock.h"
+#include <QTime>
+
+SimpleClock::SimpleClock(QObject *parent)
```

Comment:
Please inline trivially simple functions into the header.
Also, initialization is safer to do at declaration:

```
...
  : second_(0), ...
```
==> 
```
private:
  int second_ = 0;
```